packages/project-access/src/project/search.ts

```diff
@@ -338,6 +339,47 @@ async function filterExtensions(pathMap: FileMapAndCache): Promise<ExtensionResu
     return results;
 }
 
+/**
+ * Generates manifest.json from provided .library file.
+ *
+ * @param paths paths to .library file
+ * @returns library result containing virtually generate manifest.json, project path and empty manifestPath
+ */
+async function getVirtualManifest(paths: string[]): Promise<LibraryResults[]> {
+    const FileSystem = await import('@ui5/fs/adapters/FileSystem');
+    const ResourceFactory = await import('@ui5/fs/resourceFactory');
+
+    const virtualLibraries = [];
+    for (const libraryPath of paths) {
+        const projectName = 'library';
+        const virBasePath = `/resources/${projectName}/`;
+        const fs = new FileSystem.default({
+            virBasePath,
+            fsBasePath: libraryPath
+        });
+        const workspace = await ResourceFactory.createWorkspace({ reader: fs, name: 'library', virBasePath });
+        try {
+            const getProject = () => {
+                return { getVersion: () => '1.0.0' };
+            };
+            await generateLibraryManifest({
+                workspace,
+                taskUtil: { getProject },
```

Comment:
@Klaus-Keller currently `taskUtil` is mocked here, not sure if `TaskUtil` instance can be easily implemented

Reply:
Should be fine. UI5 team has agreed to implement a unit test that ensures future versions will  work if only `getVersion` is implemented